### ISSUE TITLE
zkevm-prover not starting 

### ISSUE THREAD

**Issue by DonatoJPS** [0xPolygonHermez/zkevm-node#3633]
I am trying to setup a local zkNode following the [official guide](https://docs.polygon.technology/zkEVM/get-started/setup-nodes/local-node/) in my local machine (macOS M1 Sonoma 14.4). As suggested in the docs as I am running all this in macOS, I added `platform: linux/amd64` line in `test/docker-compose.yml` file for `zkevm-prover` and `zkevm-mock-l1-network` services.

All the images are build successfully and apparently all containers are starting:
![image](https://github.com/0xPolygonHermez/zkevm-node/assets/98332680/e33db827-c819-4f51-9f4b-30e0782d2990)

But my JSON RPC node is not responding. I started to examine in detail and I found that `zkevm-prover` is not starting. If I inspect its logs, I don't see a single line: 
![image](https://github.com/0xPolygonHermez/zkevm-node/assets/98332680/e42eceb4-1d4d-4404-9409-66196b4d8e91)

And looking into `zkevm-json-rpc` node logs:
![image](https://github.com/0xPolygonHermez/zkevm-node/assets/98332680/9d5ce926-0e96-40cc-9b2e-3db32a8cc263)

The same log lines can be found in `zkevm-sync` and `zkevm-sequencer`.

It looks like `zkevm-prover` is not starting correctly. Could this issue be related to my architecture/OS configuration? Am I setting something up incorrectly?

**Comment by tclemos** [comment]
Hi @DonatoJPS, this issue is a known issue when running the `zkevm-prover` while using a M1 processor.

As you have added the `platform` to the zkevm-prover docker-compose service, try adding also the ENV var `EXPERIMENTAL_DOCKER_DESKTOP_FORCE_QEMU=1`

You should end-up with a config similar to this:

```yaml
  zkevm-prover:
    container_name: zkevm-prover
    image: hermeznetwork/zkevm-prover:v6.0.0
    platform: linux/amd64
    environment:
      - EXPERIMENTAL_DOCKER_DESKTOP_FORCE_QEMU=1
    ...
```

**Comment by DonatoJPS** [comment]
I also added that environment variable as you suggested, but that feature is available starting from Docker Desktop v4.27. I had to update my Docker Desktop version, and it worked fine!

Thanks @tclemos! Closing issue... 🙌
